README.md

```diff
@@ -1 +1,50 @@
-# rawhide-rpmdeplint-pipeline
+# rpmdeplint-pipeline
+
+This repository contains definition of the rpmdeplint CI pipeline.
+
+This pipeline runs [rpmdeplint](https://github.com/fedora-ci/rpmdeplint-image) tool on RPM packages and publishes results via [CI Messages](https://pagure.io/fedora-ci/messages).
```

Comment:
Another little one =) 

"This pipeline runs the [rpmdeplint](https://github.com/fedora-ci/rpmdeplint-image) tool "